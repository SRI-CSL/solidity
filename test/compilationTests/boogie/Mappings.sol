pragma solidity ^0.4.23;

contract Mappings {
    mapping(address=>uint) user_balances;

    function getMyBalance() view public returns (uint) {
        return user_balances[msg.sender];
    }
}