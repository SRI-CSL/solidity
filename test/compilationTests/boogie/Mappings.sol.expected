Verifying Mappings::getMyBalance: OK
Verifying Mappings::setMyBalance: OK
Verifying Nested::getSomething: OK
Verifying Nested::setSomething: OK
No errors found.
